en-US/firefox-ios.xliff

```diff
@@ -1908,13 +1918,18 @@
         <note>Title for the wallpaper onboarding page in our Onboarding screens. This describes to the user that they can choose different wallpapers.</note>
       </trans-unit>
       <trans-unit id="Onboarding.Welcome.Description.v106" xml:space="preserve">
-        <source>Firefox puts people over profits and defends your privacy by default.</source>
-        <target>Firefox puts people over profits and defends your privacy by default.</target>
+        <source>Firefox puts people over profits and defends your privacy as you browse. </source>
```

Comment:
Also seems like it's fallen through the cracks in previous reviews: it's better to use a placeholder here instead of app name "Firefox" out (and call that out in comment)